native_client/ctcdecode/ctc_beam_search_decoder.cpp

```diff
@@ -93,6 +99,15 @@ DecoderState::next(const double *probs,
       full_beam = (num_prefixes == beam_size_);
     }
 
+    // save the softmax of the current timestep
+    if(keep_logits_) {
```

Comment:
nit: space after `if`, `else`, `for`, `while`, `do`

Reply:
I'll change this one, do you want me to change others that I find?

Reply:
in your code, yes. in existing code, no.